lms/templates/learner_dashboard/new_program_details.html

```diff
@@ -0,0 +1,29 @@
+## Override the default styles_version to the Pattern Library version (version 2)
```

Comment:
This is a copy of the existing program_details.html  
The JS comes from the program details factory called here.  
We could either make a copy of that as well, or pass a flag into it to toggle display differences. 
I figured I would leave that to the next subtask which asks to add the backbone outline